src/i18n/strings/ar.json

```diff
@@ -191,6 +191,8 @@
     "Command error": "خطأ في الأمر",
     "Usage": "الاستخدام",
     "Prepends ¯\\_(ツ)_/¯ to a plain-text message": "ادخل احد الرموز ¯\\_(ツ)_/¯ قبل نص الرسالة",
+    "Prepends (╯°□°）╯︵ ┻━┻ to a plain-text message": "ادخل احد الرموز (╯°□°）╯︵ ┻━┻ قبل نص الرسالة",
```

Comment:
Hmm, it seems you've added translations, for every language, which was probably a lot of work... 😱 But actually, [Weblate](https://translate.element.io/) is the source of truth for translations, and we only add them there to avoid conflicts.

Please revert all the translation strings here and only add to the `en_EN.json` file.

Reply:
Will do!